src/modules/rtpengine/rtpengine.c

```diff
@@ -261,7 +262,7 @@ static int add_rtpp_node_info(
 		void *ptrs, struct rtpp_node *crt_rtpp, struct rtpp_set *rtpp_list);
 static int rtpp_test_ping(struct rtpp_node *node);
 
-static void rtpengine_dtmf_events_loop(int rank);
+static void rtpengine_dtmf_events_loop();
```

Comment:
nit: for better standards compliance, be explicit with `(void)`

Reply:
Fixed.